lib/web/apiserver.go

```diff
@@ -1135,30 +1098,13 @@ func (m *Handler) getSiteNodes(w http.ResponseWriter, r *http.Request, p httprou
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
-	sessions, err := clt.GetSessions(namespace)
-	if err != nil {
-		return nil, trace.Wrap(err)
-	}
-	nodeMap := make(map[string]*nodeWithSessions, len(servers))
-	for i := range servers {
-		nodeMap[servers[i].GetName()] = &nodeWithSessions{Node: *servers[i].V1()}
-	}
-	for i := range sessions {
-		sess := sessions[i]
-		for _, p := range sess.Parties {
-			if _, ok := nodeMap[p.ServerID]; ok {
-				nodeMap[p.ServerID].Sessions = append(nodeMap[p.ServerID].Sessions, sess)
-			}
-		}
-	}
-	nodes := make([]nodeWithSessions, 0, len(nodeMap))
-	for key := range nodeMap {
-		nodes = append(nodes, *nodeMap[key])
+
+	items := []interface{}{}
+	for _, server := range servers {
+		items = append(items, server.V1())
```

Comment:
I think we have a `ServerSpecV2` and we should use the latest everywhere in code.

Reply:
if possible, yes, I would like to grandfather the V1 spec of server, but not sure if it fits the scope of the change. may be create a separate issue/PR for this

Reply:
It needs to be replaced with a new UI object (/lib/web/ui) which can be constructed using interfaces. I will create a separate issue for that.